modules/microshift-backing-up-manually.adoc

```diff
@@ -0,0 +1,31 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-backing-up-manually_{context}"]
+= Backing up {product-title} data manually
+
+If you are not using an OSTree system, you can back up data manually at any time. Data backups are advised prior to any system updates. Backups are created in the `/var/lib/microshift-backups` directory by default. You can also specify a directory by running an optional backup command.
+
+.Prerequisites
+* You have root access to the host.
+* {product-title} and related workloads are stopped.
+
+.Procedure
+. Manually create a backup file using the default parent directory, `/var/lib/microshift-backups`, by running the following command:
++
+[source, terminal]
+----
+# microshift backup --storage
+----
+
+. Optional. Manually create a backup file specifying the parent directory in the <name> value by running the following command:
++
+[source, terminal]
+----
+# microshift backup --<name>
+----
++
+.Verification
+. You can verify that the backup was created by viewing the file in the specified directory.
```

Comment:
The backup is not a file, it's a directory; it's `<name>` directory in the `<storage>` directory.
For example: `/var/lib/microshift-backups/my-custom-backup/` or `/var/lib/custom-storage-location/another-custom-backup/`

So the schema is: `<storage>/<name>`